### PR TITLE
Add support for Python 3.9 & boilerplate

### PR DESCRIPTION
**Pull request by ewjoachim**
- Add support to Python 3.9
- Switch to Poetry
- Switch to a new folder named `scripts` that contains small scripts that reference usual project commands (instead of tox)
- Speed up CI

### Successful PR Checklist:

- [ ] Tests
  - [x] (not applicable?)
- [x] Documentation
  - [ ] (not applicable?)

### CODE REVIEW

**Review by mgu — APPROVED** [review]
:rocket:

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      # Drafts the next Release notes as Pull Requests are merged into "master"
+      # Drafts the next Release notes as Pull Requests are merged into "main"
```
Comment (elemoine) [inline .github/workflows/release-drafter.yml:12]: main ?

Comment (ewjoachim) [reply .github/workflows/release-drafter.yml:12]: damn, copy-paste error

Comment (ewjoachim) [reply .github/workflows/release-drafter.yml:12]:
```suggestion
      # Drafts the next Release notes as Pull Requests are merged into "master"
```

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Scripts
+
+Each individual file in this folder is a script, aimed to capture a command
+for the project.
+
+Because Makefiles have bitten the author once too much.
```
Comment (elemoine) [inline scripts/README.md:6]: I cannot accept such a statement, please remove it :-)

Comment (ewjoachim) [inline scripts/README.md:6]:
```suggestion
for the project.
```

**Review by elemoine — APPROVED** [review]
Looks fantastic! Thanks for this.

**Pull request merged**